src/Interpreters/Cluster.cpp

```diff
@@ -485,6 +487,7 @@ Cluster::Cluster(const Poco::Util::AbstractConfiguration & config,
             }
 
             Addresses shard_local_addresses;
+            Addresses all_local_addresses;
```

Comment:
But the above vector stores local addresses right?
Shouldn't this be `shard_all_addresses`?